pkg/kubefed2/disable.go

```diff
@@ -159,9 +159,11 @@ func DisableFederation(cmdOut io.Writer, config *rest.Config, typeConfigName ctl
 	}
 
 	write := func(data string) {
-		if cmdOut != nil {
-			cmdOut.Write([]byte(data))
+		if cmdOut == nil {
+			return
 		}
+
+		_, _ = cmdOut.Write([]byte(data))
```

Comment:
I apologize for not being clear.  The error should not be ignored.  It should be logged with `glog.Fatalf`.